WatsonTcp/WatsonTcpClient.cs

```diff
@@ -1226,22 +1226,18 @@ private void SendDataStream(long contentLength, string guid, Stream stream)
                     throw new ArgumentOutOfRangeException(nameof(_Settings.Encryption.Passphrase));
                 }
 
-                byte[] streamData;
+                byte[] streamData = WatsonCommon.ReadStreamFully(stream);

```

Comment:
This might be where I got into trouble in the other package I referenced re: the "read stream fully" issue in .NET 6.

Reply:
I'll be doing multiple tests on my environment with .NET6 to avoid any possible trouble.